test/e2e/resource-filtering/filtering.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filtering
+
+import (
+	"context"
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/vmware-tanzu/velero/test/e2e"
+	. "github.com/vmware-tanzu/velero/test/e2e/test"
+	. "github.com/vmware-tanzu/velero/test/e2e/util/k8s"
+	veleroutils "github.com/vmware-tanzu/velero/test/e2e/util/velero"
+)
+
+/*
+Includes
+	include-namespaces
+	include-resources
+	include-cluster-resources
+	selector
+Excludes
+	exclude-namespaces
+	exclude-resources
+velero.io/exclude-from-backup=true
+*/
+
+func ResourceFilering(test VeleroTest, isTestBackup bool) func() {
```

Comment:
Typo: ResourceFilering -> ResourceFiltering

Move the function `ResourceFiltering` into package `e2e/test` and rename it because it is a general function used not only for resource filtering test cases. Correct me if I'm wrong